Prox/Prox/Widgets/ReviewContainerView.swift

```diff
@@ -106,6 +115,8 @@ class ReviewContainerView: UIView {
 
         constraints += [bottomAnchor.constraint(equalTo: numberOfReviewersLabel.bottomAnchor)]
 
+        collapsedConstraints = [ heightAnchor.constraint(equalToConstant: 0) ]
```

Comment:
If we set a height constraint of 0, then disable it, is the view going to want to stay size 0? Or does it go back and use the intrinsic content size again? I'm assuming the latter, but just want to make sure you check this case!